app/hotels/src/singleHotel/HotelDetailScreen.js

```diff
@@ -86,10 +87,20 @@ export class HotelDetailScreen extends React.Component<
     const isIOS = Platform.OS === 'ios';
     const isNarrow = this.isNarrowLayout();
     const hasScrolledPastImage = e.nativeEvent.contentOffset.y > 180;
-    if (isIOS && isNarrow && hasScrolledPastImage) {
-      StatusBar.setBarStyle('dark-content');
-    } else if (isIOS && isNarrow && !hasScrolledPastImage) {
-      StatusBar.setBarStyle('light-content');
+    if (
+      isIOS &&
```

Comment:
Maybe have
```
const shouldBeDark = isIOS && isNarrow && hasScrolledPastImage && this.state.barStyle !== 'dark-content';
```
and similarly for the second one, to make it a bit more readable?
This is just a suggestion.